tests/LabelAssertionTests.py

```diff
@@ -0,0 +1,63 @@
+import unittest
+import json
+from PullRequest import PullRequest
+
+NO_LABELS = ['']
```

Comment:
shouldn't this be an empty list? If `ConfigParser` is returning a list with a single empty item, I think we should add a check `or None` or `or []` so that the result is a little nicer to work with. Currently if we were to do length checks this would return "1" which would be confusing.

Reply:
Hm yea that's fair, I had actually just changed this to make the parsing a little nicer, but you're probably right that that's not worth the trade-off. Will change